packages/sdk/schema/src/projection.ts

```diff
@@ -44,6 +44,7 @@ export class ViewProjection {
   private _fieldProjections = computed(() => this._view.fields.map((field) => this.getFieldProjection(field.id)));
   private _hiddenProperties = computed(() => this._view.hiddenFields?.map((field) => field.path as string) ?? []);
 
+  // TOOD(burdon): Should this take an instane of S.S.AnyNoContext and derive the JsonSchemaType itselft (and watch for reactivity)?
```

Comment:
Maybe, but we manipulate the underlying `JsonSchema` directly in a few cases here. If we derive `JsonSchema` and only take `EffectSchema` then we need to make sure the changes get pushed back into the underlying `JsonSchema`.